lib/dom-event-delegate.js

```diff
@@ -50,6 +60,12 @@ Box.DOMEventDelegate = (function() {
 		// Without checking the existence of a parentNode and returning null, we would throw errors
 		while (!found && element && !isModuleElement(element)) {
 			element = element.parentNode;
+
+			// Stop if we hit a non-basic HTML element such as the DocumentNode
+			if (!isBasicElement(element)) {
+				break;
```

Comment:
This means any element that doesn't have a parentNode is always a non-HTMLElement, correct?